react/features/toolbox/components/VideoMuteButton.js

```diff
@@ -136,10 +135,13 @@ class VideoMuteButton extends AbstractVideoMuteButton<Props, *> {
             this.props.dispatch(
                 setAudioOnly(false, /* ensureTrack */ true));
         }
+        const tracks = APP.store.getState()['features/base/tracks'];
```

Comment:
this won't work on mobile (there is no APP). Extract the mediaType in `_mapStateToProps`

Reply:
ok, made the change.